### ISSUE TITLE
Same configuration but different results when using pretrained model 'slowfast_kinetics_pretrained_r50_4x16x1_20e_ava_rgb_20201217-6e7c704d.pth' to inference.

### ISSUE THREAD

**Issue by Wangxunhang** [open-mmlab/mmaction2#2606]
### Branch

main branch (1.x version, such as `v1.0.0`, or `dev-1.x` branch)

### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmaction2/issues) and [Discussions](https://github.com/open-mmlab/mmaction2/discussions) but cannot get the expected help.
- [X] I have read the [documentation](https://mmaction2.readthedocs.io/en/latest/) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmaction2).

### Environment

mmaction2              0.24.1          
mmcv-full              1.7.0
mmdet                  2.28.2
mmengine               0.7.2

### Describe the bug

When I run 'webcam_demo_spatiotemporal_det.py' for behavior recognition on the same video, the results of each inference are very different. I set the label_map as a single label, sometimes the average score is 0.9, and sometimes it is only 0.6. What is the reason?

Tips:

slowfast uses the pre-trained model 'slowfast_kinetics_pretrained_r50_4x16x1_20e_ava_rgb_20201217-6e7c704d.pth',

fastrcnn uses the pre-trained model 'faster_rcnn_r50_fpn_2x_coco_bbox_mAP-0.384_20200504_210434-a5d8aa15.pth'
